R/get_gene_expression.R

```diff
@@ -231,52 +221,39 @@ get_gene_expression = function(these_samples_metadata,
       
       # add column `multi_exp` to inform whether there are more than one 
       # `mrna_sample_id` associated to a `sample_id`
```

Comment:
It seems like the whole expression data set is loaded in before anything is done with the metadata. This seems wasteful but perhaps it's a limitation imposed by having some of the required information exist only in the expression table?